virt/virt-web-console-overview.adoc

```diff
@@ -347,43 +347,50 @@ By default, Red Hat templates are stored in the `openshift` project. You can edi
 |====
 =====
 
-// .*InstanceTypes* tab
-// [%collapsible]
+[id="virtualization-instancetype_{context}"]
+=== InstanceTypes tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+
+:FeatureName: Creating a virtual machine from an instance type
+include::snippets/technology-preview.adoc[]
+
+|====
+|Element
+|Description
 
-// =====
-// [cols="1,3a", options="header"]
-// |====
-// |Element
-// |Description
+|*Volumes project*
+|The current namespace.
 
-// |*Volumes project* list
-// |Select the volume to boot your VM from.
+|*Select volume to boot from*
```

Comment:
This might change, but you don't have to worry about it right now. I think I saw a discussion about the wording.